genesyscloud/resource_genesyscloud_routing_settings.go

```diff
@@ -0,0 +1,277 @@
+package genesyscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v75/platformclientv2"
+	"github.com/mypurecloud/terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+	"log"
+)
+
+func resourceRoutingSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: "An organizations routing settings",
+
+		CreateContext: createWithPooledClient(createRoutingSettings),
+		ReadContext:   readWithPooledClient(readRoutingSettings),
+		UpdateContext: updateWithPooledClient(updateRoutingSettings),
+		DeleteContext: deleteWithPooledClient(deleteRoutingSettings),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"reset_agent_on_presence_change": {
```

Comment:
This may need to be computed. Try creating a test without providing this attribute. It would probably error out when it doesn't expect this to have a value